src/list.tsx

```diff
@@ -274,11 +276,21 @@ interface BasicRowContentProps extends CommonProps {
     atomicReading?: boolean;
 }
 
+type SwitchDisclosure = {
+    expanded: boolean; // is the related content expanded
+    controlsId?: string; // id of the related content (if any)
+    live?: 'off' | 'polite' | 'assertive'; // announcement channel when changing expanded
+    busy?: boolean; // blocks UI and sets aria-busy
+    showSpinner?: boolean; // shows spinner on the right
```

Comment:
`showSpinner` and `busy` aren't the same?

Reply:
one was for showing the spinner the other for setting aria-busy, but as the spinner is not in the spec, i'll delete the showSpinner